.changeset/shiny-lions-call.md

```diff
@@ -0,0 +1,15 @@
+---
+'@twilio-paste/alert': patch
```

Comment:
Are these minor or patch? Asking because I think this results in a visual change, value changes for the tokens?

Reply:
These are just visual changes, we swapped some of the tokens for other ones. Would that be minor or patch?

Reply:
Patch for me. We're just tweaking existing things visually. There's no extra functionality or changes in functionality. 

Radio might be the only one that's minor as it has a new element you can customize.